.circleci/config.yml

```diff
@@ -78,6 +79,16 @@ jobs:
           name: Build React App
           command: |
             yarn build
+      - run:
+          name: Test React App
+          command: |
+            yarn test
+      - run:
+          name: Test React App Codecover
+          command: |
+            yarn run test:coverage
+      - codecov/upload:
+          file: react/coverage
```

Comment:
this should be a hidden folder (react/.coverage).
At the moment it does not upload data.

Reply:
I noticed that the .gitignore file in the react folder contains `/coverage`, so I thought it would be fine to name it like that. However, I also understand that it would be nice for the formatting to be consistent with what's written in the back-end. Is `react/.coverage` is preferable to `react/coverage`?

Not sure if there is a difference between a hidden folder and an ignored folder.

Reply:
A folder (more general: any file) can be hidden in the context of the operating system directory view I reckon. E.g. on Linux the standard directory view `ls` does not show hidden files (such starting with a period) but `ls -a` does.
Any file can be ignored in the context of git (probably version control in general) meaning that the file is not taken into account for version control by git. It is irrelevant whether the file is hidden or not, git only cares about the filename and whether it matches a pattern in `.gitignore`.

So far about the terms in general :laughing: For the main question I'd propose that the coverage data file should be ignored since in a codebase it's mostly the code files and configuration files that are being cared about. It's consistent to follow the filename convention of the back-end. 
tldr rename the coverage data file into `.coverage` and add `.coverage` to ´.gitignore` :)

Reply:
@pylipp Thanks for the clarification! As of now, the `codecov/upload` step looks like this:
```
- codecov/upload:
    file: coverage/clover.xml
```
I am using `coverage` instead of `.coverage` because that is what `yarn run test:coverage` generates. In order to use `.coverage`, we will have to find a way to change what the command generates.

Reply:
Currently, Codecov indicates that it is able to read `coverage/clover.xml`, but we are getting an HTTP 400 error. It says that the "Provided token is not a UUID." You can see the error that we are getting [here](https://app.circleci.com/pipelines/github/boxwise/boxtribute/240/workflows/cb158d04-2048-4ef0-9a57-a0e5fe5f54a5/jobs/1042) under "Upload Coverage Results".